Area.cpp

```diff
@@ -0,0 +1,15 @@
+#ifndef Area_cpp
+#define Area_cpp
+
+#include <iostream>
+
+
+using namespace std;
+
+class Area
+{
+public:
+    virtual int CalculateArea() = 0;
```

Comment:
Widely accepted convention to write function names is camel-case with first letter always small.